src/main/java/fr/minuskube/inv/SmartInventory.java

```diff
@@ -13,26 +13,97 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Base event for creating SmartInvs.
+ */
 @SuppressWarnings("unchecked")
 public class SmartInventory {
-
+    /**
+     * ID string. The field is the unique id for the custom inventory.
+     */
     private String id;
+    /**
+     * Title String. The field is the title for the custom inventory.
+     */
     private String title;
+    /**
+     * Inventory type. The field is the inventory type. The inventory type
+     * may be changed to any of the following:
+     * <ul>
+     *       <li>Chest</li>
+     *       <li>Dispenser</li>
+     *      <li>Dropper</li>
+     *      <li>Furnace</li>
+     *      <li>Workbench</li>
+     *      <li>Crafting</li>
+     *      <li>Enchanting</li>
+     *      <li>Brewing</li>
+     *      <li>Player</li>
+     *      <li>creative</li>
+     *      <li>Merchant</li>
+     *      <li>Ender Chest</li>
+     *      <li>Anvil</li>
+     *      <li>Beacon</li>
+     *      <li>Hopper</li>
+     *  </ul>
+     */
     private InventoryType type;
+    /**
+     * Rows &amp; Columns Integer. This field has default values for both rows (6) &amp; columns (9).
+     */
     private int rows, columns;
     private boolean closeable;
-
+    /**
+     * Inventory close boolean. This field being a boolean, is the key to allowing the inventory to
+     * be closed or not.
+     */
```

Comment:
This JavaDoc is just floating around with nothing attached to it

Reply:
Fixed the floating JavaDoc. Also added some more Javadocs along with clearing up the meaning behind what things are actually suppose to do. Will add more when I can, with whatever you have not added in v2.0.0